projects/observability/src/shared/components/radar/tooltip/radial-data-lookup-strategy.ts

```diff
@@ -15,7 +15,7 @@ export class RadialDataLookupStrategy {
 
   public constructor(private readonly allSeries: RadarSeries[], radialAxisData: RadarAxisData[]) {
     this.radialBisector = bisector(axisData => axisData.axisRadian);
-    this.radialAxisData = clone(radialAxisData);
+    this.radialAxisData = clone(radialAxisData.sort(axisData => axisData.axisRadian));
```

Comment:
please sort the cloned array instead of the argument

Reply:
Out of curiosity, if this data is not sorted why is this not an issue in other browsers?

Reply:
It is sorted in other browsers like chrome and safari.
Without the fix, in chrome radialAxisData, is sorted by axisRadian
<img width="1275" alt="Screenshot 2021-07-15 at 10 51 29 AM" src="https://user-images.githubusercontent.com/86998924/125733495-392f80a5-272d-4e14-a3fe-b9e89f7adce7.png">


The same code in firefox returns  radialAxisData is not sorted
<img width="1275" alt="Screenshot 2021-07-15 at 10 52 19 AM" src="https://user-images.githubusercontent.com/86998924/125733662-c2d5d027-7f7e-41f5-a702-d9c2c93e4061.png">

Reply:
oh strange - thanks for explaining!